test/studies/shootout/submitted/CompatGMP.chpl

```diff
@@ -15,10 +15,10 @@ proc ref bigint.sub(x, y) do BigInteger.sub(this, x, y);
 proc ref bigint.mul(x, y) do BigInteger.mul(this, x, y);
 
 @deprecated("bigint.addmul method is deprecated - please use the standalone function addMul instead")
-proc ref bigint.addmul(x, y) do BigInteger.addmul(this, x, y);
+proc ref bigint.addmul(x, y) do BigInteger.addMul(this, x, y);
 
 @deprecated("bigint.submul method is deprecated - please use the standalone function subMul instead")
-proc ref bigint.submul(x, y) do BigInteger.submul(this, x, y);
+proc ref bigint.submul(x, y) do BigInteger.subMul(this, x, y);
```

Comment:
This might be the right thing to do here but just making sure; do we really want to change the this file since it's under the `shootout/submitted` directory?

I think historically we leave the deprecated functions in there, and only remove them when we update the shootout? I don't think it's a big deal in this case however.

Reply:
Yes, this is a compatibility module that we add the deprecated function to when removing the function. Otherwise, removing the function would break these tests.